dotcom-rendering/src/web/components/FixedSmallSlowIV.tsx

```diff
@@ -0,0 +1,57 @@
+import { ArticleDesign } from '@guardian/libs';
+import { Card } from './Card/Card';
+import { LI } from './Card/components/LI';
+import { UL } from './Card/components/UL';
+
+type Props = {
+	trails: TrailType[];
+};
+
+export const FixedSmallSlowIV = ({ trails }: Props) => {
+	const slicedTrails = trails.slice(0, 4);
+
+	return (
+		<UL direction="row">
+			{slicedTrails.map((trail, index) => {
+				return (
+					<LI
+						percentage="25%"
```

Comment:
I think you can remove this line and it should default to evenly distributing the cards

```suggestion
```

Reply:
Thanks!